docs/resiliency/index.asciidoc

```diff
@@ -170,6 +170,17 @@ shard.
 
 == Completed
 
+[float]
+=== Replicas can fail to replicate delete-and-reindex operations during certain network partitions (STATUS: DONE, V6.0.0)
```

Comment:
delete-and-reindex sounds too much related to the reindex API, maybe come up with a better name?

Reply:
I also think it makes it sound like a single operation when it is not.